README.md

```diff
@@ -31,10 +32,115 @@ sudo kubeadm join 10.10.1.1:6443 --token btytkp.7nh8pawcdsi23g4x \
 	--discovery-token-ca-cert-hash sha256:9d1802d5451e559b5c076db6901865b164bd201ed46ce38c1cba03e89618e027 \
   --cri-socket unix:///var/run/cri-dockerd.sock
 ```
+**Note:** add config from *master* node to *worker* node.
```

Comment:
What is the purpose of Line#35 to Line#44? This is for running kubectl on worker node?

Reply:
When I was first deploying flame and I used kubectl get nodes to check the cluster deployment there would be an error. Now that you mention that it is only for running kubectl on worker node I will check if we do not need this and we can just run kubectl on master node.

Reply:
We do not need this, I have removed it from the readme.